backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlSchemaMetadataExecutor.java

```diff
@@ -124,14 +124,14 @@ private static void updateMembershipForUser(
   }
 
   static String getRolePrefix(String name) {
-    return Constants.MG_ROLE_PREFIX + name.toUpperCase() + "/";
```

Comment:
same for roles , PR should not only clearly indicate that this change is breaking of should include details on what breaks and how it can be 'fixed'